pkgs/development/tools/ocaml/ocaml-lsp/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub, ocamlPackages }:
+
+with ocamlPackages; buildDunePackage rec {
```

Comment:
Does the version of OCaml used for compiling this LSP server matter? Should it be the same as the version used for development?

For merlin, e.g., it does matter and this is why the merlin package is inside `ocamlPackages` rather than taking it as argument.

Reply:
I haven't tested thoroughly with different OCaml versions, but IIUC it may depend on the Ocaml version